.github/workflows/upload-artifacts.yml

```diff
@@ -1,8 +1,6 @@
 name: "Generate and upload distributions"
 on:
-  push:
-    branches:
-      - release/**
```

Comment:
enabled temporarily for testing

Reply:
Works well, the distribution validation task also passed which means it most likely looks good

https://github.com/getsentry/sentry-kotlin-multiplatform/actions/runs/16355042742